poly-synth/src/poly-synth.js

```diff
@@ -13,80 +13,106 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
+/**
+ * @class PolySynth
+ * @description Manages the life cycle of voices.
+ */
 class PolySynth {
   /**
-   * @class  PolySynth
    * @constructor
-   * PolySynth manages the life cycle of voices.
-   * @param {AudioContext} context the audio context
+   * @param {AudioContext} context The audio context.
    */
   constructor(context) {
     if (!(context instanceof AudioContext))
       throw context + ' is not a valid audio context.';
 
     this.context_ = context;
-    this.sustainedVoices_ = [];
+    this.activeVoices_ = [];
     this.releasedVoices_ = [];
 
-    // The default maximum cutoff is set near the upper edge of human hearing
-    // and the minimum cutoff is near the lower edge of human hearing.
+    // The default maximum cutoff is set below the upper edge of human
+    // hearing and the minimum cutoff is above the lower edge of human hearing.
```

Comment:
Dunno about you, but my old ears can still hear 12 kHz tones.  Update the comment or the limit.